lib/pg_party/adapter_decorator.rb

```diff
@@ -265,7 +271,7 @@ def recursive_add_index(table_name:, index_name:, index_type:, index_columns:, i
       # If this is a partitioned table, add index ONLY on this table.
       if table_partitioned?(table_name)
         add_index_only(table_name, type: index_type, name: updated_name, using: using, columns: index_columns,
-                       options: index_options)
+                                   options: index_options)
```

Comment:
I think each keyword arg should be on its own line to look like the `create_table_like` calls above

Reply:
Completely agree, updated.